rest-api/tools/ehr_upload.py

```diff
@@ -35,13 +36,14 @@ def get_participants_under_sites(session, organization, source_bucket, destinati
           participant) + "/* " + "gs://" + destination_bucket + "/Participant/" + \
                  google_group + "/P" + str(participant) + "/"
 
-        system_call = subprocess.call(gsutil, shell=True)
-        if system_call == 0:
-          print "Successfully moved folder " + google_group + '/' + str(participant)
-        elif system_call == 1:
-          print "There was an error moving folder " + google_group + '/' + str(participant)
+        gsutil = shlex.split(str(gsutil))
```

Comment:
Starting with a string and then parsing is a bit roundabout, since you define the string. I'd just define the process arguments as an array to begin with.